File: request.js
//Method 1: Using XttpRequest web browser object (Node js implementation: Xhr2)
const request = new XMLHttpRequest();
request.open('GET', 'https://www.example.com/get-data');
request.send();
request.onload = () => {
    if (request.status === 200) {
        var data = JSON.parse(request.response).data;
        data.sort(function(a, b) {
            return a.id - b.id;
        })
        data.forEach((item, index) => {
            console.log(item.id);
            console.log(item.name)
        })
    } else {
        console.log(`error ${request.status}`);
    }
}

File: request.py
import requests
import json
req = requests.get('https://www.example.com/get-data')
res = json.loads(req.text)
item_array = res['data']
item_array.sort(key=lambda x: x["id"])
for items in item_array:
    print(items.id)
    print(items.item)